.github/workflows/ci.yml

```
@@ -145,6 +150,7 @@ jobs:
         run: nix develop -L --no-update-lock-file --command just build-cli
 
       - name: "CLI Unit Tests"
+        if: ${{ matrix.test-tags == 'containerize' }}
```

only running these tests in `containerize` is not obvious.
optimally we'd have a run for only uni tests.
For the moment it gets the job done I guess.

---

NB. I changed this to 
```suggestion
        # Only run unit tests if when not running specialized integration tests
        if: ${{ matrix.test-tags == '!activate,!containerize,!catalog' }}
```